depthai_sdk/src/depthai_sdk/visualize/visualizer.py

```diff
@@ -199,55 +200,129 @@ def serialize(self) -> str:
 
         return json.dumps(parent, cls=JSONEncoder)
 
-    def output(self, **kwargs: dict) -> 'Visualizer':
+    def reset(self):
+        self.objects.clear()
+
+    def output(self,
+               img_scale: float = None,
+               show_fps: bool = None) -> 'Visualizer':
         """
         Configure the output of the visualizer.
 
         Args:
-            **kwargs:
+            img_scale: Scale of the output image.
+            show_fps: Flag that indicates if the FPS should be shown.
 
         Returns:
             self
         """
-        self.config.output = replace(self.config.output, **kwargs)
+        kwargs = locals()
+        kwargs.pop('self')
+        kwargs = {k: v for k, v in kwargs.items() if v is not None}
```

Comment:
Maybe have a function for this logic, so it's not copy pasted across all public functions?